.github/workflows/on-release.yaml

```diff
@@ -44,11 +46,27 @@ jobs:
           # Those targets are only used on macOS runners so it's in an `if` to slightly speed up windows and linux builds.
           targets: ${{ matrix.platform == 'macos-latest' && 'aarch64-apple-darwin,x86_64-apple-darwin' || '' }}
 
-      - name: install dependencies (ubuntu)
-        if: matrix.platform == 'ubuntu-22.04' # This must match the platform value defined above.
+      - name: install dependencies
```

Comment:
You should keep the mention about Ubuntu. Leave it to "install dependencies (Linux)" something like that

Reply:
I've added the Linux motion